src/core/AppRouter/AdminSubRouter.tsx

```diff
@@ -25,6 +25,8 @@ import { WorldsDashboard } from "pages/WorldsDashboard";
 
 import { AdminVenueView } from "components/organisms/AdminVenueView";
 
+import "scss/adminGlobal.scss";
```

Comment:
```suggestion
import "scss/admin.scss";
```

Two notes, though:

1. Should not be here. Router has too many concerns already.

    This import should be made at page level.
    Assuming all admin pages would have a single framing layout,
    that's only one component the file should be included it.

    Example: if both `WorldsDashboard` and `SpacesDashboard` components have `AdminLayout` one, then only   `AdminLayout` needs to import the SCSS file.

2. import may have time dimension, not just spatial one, what would the issue/solution be in that case?
Example:
   1. Navigate to attendee URL (no style added to DOM)
   2. Navigate to admin URL (the admin styles added to the DOM)
   3. Navigate back to the attendee URL (are the admin styles still part of the DOM?)

Reply:
2. Just checked and this is a problem. I'll see if there's another way to do this.